### PR TITLE
Misc Queries

### PR DESCRIPTION
**Pull request by nca6109**
### Description
One new query added: `getAllTrunkSets`
-----
##### Description
Get all "trunk" panel sets in the database
##### Params
None
##### Returns
`PanelSet[]`

### CODE REVIEW

**Review by BlckHawker — COMMENTED** [review]
### getStatus
Maybe I didn't test the function enough, but I don't see the purpose of both `message` and `linked` as they seem to be showing the same information. I would remove `message` as that information of which `panel_set` the hook attaches to can be found by getting the `hook`.

### Other
If I recall correctly, the other teams wanted to have a method that would get a tree given the root (`panel_set`), but it's not necessarily a trunk. Will that be added here as well, or did we assign that to someone else?

**Comment by nca6109** [comment]
> ### getStatus
> Maybe I didn't test the function enough, but I don't see the purpose of both `message` and `linked` as they seem to be showing the same information. I would remove `message` as that information of which `panel_set` the hook attaches to can be found by getting the `hook`.
> 
> ### Other
> If I recall correctly, the other teams wanted to have a method that would get a tree given the root (`panel_set`), but it's not necessarily a trunk. Will that be added here as well, or did we assign that to someone else?

I'll change `getStatus` to just a Boolean return. And getting a tree from a root was a separate task on the board so I will make that query in a later request.

File: src/tests/hook.test.ts [hunk 1]
```diff
@@ -163,3 +163,22 @@ describe('Add Set To Hook Controller', () => {
...
+    });
+
+    test('If an something goes wrong, an error should be returned', async () => {
+        (hookService.getStatus as jest.Mock).mockReturnValue(() => { throw new Error('Error Message'); });
+
+        const response = await _getStatusController(sequelizeMock())(1);
+
+        expect(response).toBeInstanceOf(Error);
+    });
+});
```
Comment (BlckHawker) [inline src/tests/hook.test.ts]:
```suggestion
```

File: src/tests/hook.test.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import {
-    _createHookController, _getHookController, _getPanelHooksController, _addSetToHookController
+    _createHookController, _getHookController, _getPanelHooksController, _addSetToHookController, _getStatusController
```
Comment (BlckHawker) [inline src/tests/hook.test.ts]:
```suggestion
    _createHookController, _getHookController, _getPanelHooksController, _addSetToHookController
```

**Review by BlckHawker — COMMENTED** [review]
Some excess status stuff, but other than that, you're good

File: src/tests/hook.test.ts (hunk 2 above)
Comment (nca6109) [reply src/tests/hook.test.ts]: Pull most recent, I think I pushed right as you started reviewing oops

**Review by BlckHawker — APPROVED** [review]
sonic-boom-knuckles

**Pull request merged**